### PR TITLE
 fix(deps): ran `npm audit fix` and updated mocha version

### PR DESCRIPTION
**Pull request by MendeNajdov**

### CODE REVIEW

**Comment by MendeNajdov** [comment]
@dainis please let me know what you think. This is ready to go as far as I'm concerned.

File: package-lock.json [hunk 1]
```diff
@@ -1173,24 +1016,46 @@
...
       "requires": {
         "define-properties": "^1.1.3",
-        "function-bind": "^1.1.1"
+        "es-abstract": "^1.17.5",
+        "string.prototype.trimend": "^1.0.0"
+      }
+    },
+    "string.prototype.trimstart": {
+      "version": "1.0.0",
+      "resolved": "http://npm.oss.360insights.io/string.prototype.trimstart/-/string.prototype.trimstart-1.0.0.tgz",
```
Comment (dainis) [inline package-lock.json:1053]: npm.oss.360insights.io ?

that isn't public repository

Comment (MendeNajdov) [reply package-lock.json:1053]: sorry I thought i removed all those.

Comment (MendeNajdov) [reply package-lock.json]: Updated

**Comment by adnanrahic** [comment]
Any updates here? I just came here to submit a PR for fixing the same issue. But, I see you beat me to it. :smile: :+1:

**Comment by otisg** [comment]
Me, too.  Might also address #42 ?

**Comment by MendeNajdov** [comment]
any progress on this?

**Comment by MendeNajdov** [comment]
@dainis please if you have an opportunity to review the change.  I've fixed the package-lock and the tests are all passing. Let me know if any additional changes are needed.

**Comment by MendeNajdov** [comment]
@dainis this is my monthly request that you please merge...Or let me know why we can't merge it.

**Comment by otisg** [comment]
@MendeNajdov maybe it's time to fork and start maintaining the fork because it looks like @dainis may be MIA?

**Comment by otisg** [comment]
Thank you, @dainis, much appreciated!

**Comment by otisg** [comment]
Oh, and do you know when you will be releasing the new version with this fix?  Need it for our dependencies over in https://github.com/sematext/sematext-agent-express .

**Pull request merged**